app/controllers/shopping_cart_controller.rb

```diff
@@ -0,0 +1,13 @@
+class ShoppingCartController < ApplicationController
+    def add_to_cart
+
+      if cookies['cart_id']
+        @cart = ShoppingCart.find_or_create_by(id: session[:cart_id])
```

Comment:
Also, on line 4 you are looking for a cookie and here you are getting it from a `session`, why?